src/ToricVarieties/NormalToricVarieties/properties.jl

```diff
@@ -152,7 +152,7 @@ false
 """
 function hastorusfactor(v::AbstractNormalToricVariety)
     return get_attribute!(v, :hastorusfactor) do
-        return pm_object(v).FAN_DIM < pm_object(v).FAN_AMBIENT_DIM
+                return Polymake.common.rank(rays(fan(v))) < ambient_dim(fan(v))
```

Comment:
Why the whitespace change? (not that it matters / is important, feel free to ignore)
```suggestion
        return Polymake.common.rank(rays(fan(v))) < ambient_dim(fan(v))
```

Reply:
That was not intended. Let me check.

Reply:
Ok. This should be fixed now.